cudamapper/src/index_generator_gpu.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#pragma once
+
+#include <cstdint>
+#include "cudamapper/index_generator.hpp"
+#include "cudamapper/sequence.hpp"
+#include "cudamapper/types.hpp"
+#include "minimizer.hpp"
+
+namespace claragenomics {
```

Comment:
in our other modules we've been adding another layer of namespace for the module itself
e.g.
```
namespace claragenomics {
namespace cudamapper {
....
}
}
```

Reply:
Ok, we'll fix that in a separate PR.
@vellamike FYI